test/Chainweb/Test/Pact/PactExec.hs

```diff
@@ -143,14 +144,14 @@ testReq6 = TestRequest
 -- -------------------------------------------------------------------------- --
 -- Utils
 
-execTest :: (forall a . (PactDbEnv' -> PactServiceM cas a) -> IO a) -> TestRequest -> ScheduledTest
+execTest
+    :: (forall a . (PactDbEnv' -> PactServiceM cas a) -> IO a)
+    -> TestRequest
+    -> ScheduledTest
```

Comment:
The linting in these test files is drive-by cleanups done while investigating a bug.